api/swagger.yml

```diff
@@ -1632,8 +1632,47 @@ components:
       required:
         - installation_id
         - reports
-
-
```

Comment:
I actually like the blank lines, it's very hard for me to udnerstand where an object ends without them